### PR TITLE
CAR-132 feat: Added dashboard user's data and leaderboard personal ranking

### PR DESCRIPTION
**Pull request by luferupa**
- Added dashboard user's data section endpoint
- Added leaderboard personal ranking with user's position endpoint
- Organization schema created
- Small User schema changes for pointsHistory

### CODE REVIEW

File: src/leaderboard/leaderboard.service.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  getCurrentMonthPoints(pointsHistory: any): number {
+    if (pointsHistory instanceof Map) {
+      return pointsHistory.get(this.key);
+    }
+    return pointsHistory[this.key];
+  }
+
+  async getPersonalRankingList(): Promise<RankedUser[]> {
+    const users = await this.userService.listUsers();
```
Comment (KariGarcia) [inline src/leaderboard/leaderboard.service.ts:35]: wouldnt this get all the users? or are we already filtering by the users in the same organization?

**Review by KariGarcia — APPROVED** [review]
LGTM 👍

**Review by KariGarcia — APPROVED** [review]
LGTM 👍

**Review by alerayes — APPROVED** [review]

File: src/leaderboard/leaderboard.service.ts (hunk 1 above)
Comment (luferupa) [reply src/leaderboard/leaderboard.service.ts:35]: Yes, this is getting all the users. As we only have 1 organization right now I let it this way. But I'll add another method in UserService to filter by organization later.

**Pull request merged**